src/wp-includes/class-walker-nav-menu.php

```diff
@@ -204,15 +242,8 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 		 * @param stdClass $args      An object of wp_nav_menu() arguments.
 		 * @param int      $depth     Depth of menu item. Used for padding.
 		 */
-		$atts = apply_filters( 'nav_menu_link_attributes', $atts, $menu_item, $args, $depth );
-
-		$attributes = '';
-		foreach ( $atts as $attr => $value ) {
-			if ( is_scalar( $value ) && '' !== $value && false !== $value ) {
-				$value       = ( 'href' === $attr ) ? esc_url( $value ) : esc_attr( $value );
-				$attributes .= ' ' . $attr . '="' . $value . '"';
-			}
-		}
+		$atts 		= apply_filters( 'nav_menu_link_attributes', $atts, $menu_item, $args, $depth );
```

Comment:
```suggestion
		$atts       = apply_filters( 'nav_menu_link_attributes', $atts, $menu_item, $args, $depth );
```
Should fix this PHPCS failure.